compiler/ballerina-parser/src/main/java/io/ballerina/compiler/internal/parser/ParserMode.java

```diff
@@ -57,9 +57,12 @@ public enum ParserMode {
     XML_CDATA_SECTION,
 
     // RegExp Parser
+    REGEXP,
```

Comment:
let's move this to `// Ballerina Parser` section since this mode is not belonging RegExp parser/lexer